internal/schemas/dto_schema.go

```diff
@@ -158,3 +158,15 @@ type ChangePasswordRequest struct {
 type RefreshTokenRequest struct {
 	RefreshToken string `json:"refreshToken" validate:"required"`
 }
+
+type FollowingDTO struct {
+	SubscriptionId uuid.UUID `json:"subscriptionId"`
+	CreationDate   string    `json:"creationDate"`
+	Following      AuthorDTO `json:"following"`
```

Comment:
```suggestion
	User AuthorDTO `json:"user"`
```

Finde ich persönlich schöner mit Following, aber du hattest heute User geschrieben. Würde das aber eher in Postman ändern mit Einstimmung der anderen